features/creating-a-manual-as-a-gds-editor.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Creating and editing manuals
+  As a GDS editor
+  I want to create and edit manuals of my own and other organisations
```

Comment:
This isn't quite accurate: GDS editors can't create new manuals which belong to other organisations, they can only view and edit existing ones.